src/ucs/debug/log.c

```diff
@@ -35,7 +35,7 @@ const char *ucs_log_level_names[] = {
 static unsigned ucs_log_handlers_count = 0;
 static ucs_log_func_t ucs_log_handlers[UCS_MAX_LOG_HANDLERS];
 static int ucs_log_initialized         = 0;
-static char ucs_log_hostname[256]      = {0};
+static char ucs_log_hostname[HOST_NAME_MAX] = {0};
```

Comment:
pls, align:
```
static int ucs_log_initialized              = 0;
static char ucs_log_hostname[HOST_NAME_MAX] = {0};
...
```

Reply:
Is this correct alignment? Spcially, `static ucs_log_func_t ucs_log_handlers[UCS_MAX_LOG_HANDLERS];`  part. It is longer than `=` on another lines.

Reply:
I didn't notice this. It should go right after variables/constants w/ default value set, because it doesn't have default value, i.e.:
```
...
static pthread_spinlock_t threads_lock      = 0;
static pthread_t threads[128]               = {0};
static ucs_log_func_t ucs_log_handlers[UCS_MAX_LOG_HANDLERS];
```

Reply:
Alined like the below. please take a look again.

```C
static unsigned ucs_log_handlers_count      = 0;
static int ucs_log_initialized              = 0;
static char ucs_log_hostname[HOST_NAME_MAX] = {0};
static int  ucs_log_pid                     = 0;
static FILE *ucs_log_file                   = NULL;
static int ucs_log_file_close               = 0;
static unsigned threads_count               = 0;
static pthread_spinlock_t threads_lock      = 0;
static pthread_t threads[128]               = {0};
static ucs_log_func_t ucs_log_handlers[UCS_MAX_LOG_HANDLERS];
```

Reply:
was it updated? I don't see those changes

Reply:
Sorry, I've pushed again. Please take a look again.

Reply:
> Sorry, I've pushed again. Please take a look again.

thanks 👍